test/unit/app/models/edition/organisations_test.rb

```
@@ -43,6 +43,34 @@ class Edition::OrganisationsTest < ActiveSupport::TestCase
     end
   end
 
+  # In: whitehall/test/unit/app/models/edition/organisations_test.rb
+  test "should remain publishable when linked organisations are invalid" do
```

Oooh, so we DON'T want to surface these errors at all. Ah well, then no need to worry about showing better association errors as I was saying 😅 

There was some secret value in surfacing the invalid orgs in that it would create a healthy self-fix cycle as users created drafts under that organisation, but that sounds more like a pain in the neck :)